src/player.h

```diff
@@ -187,7 +187,7 @@ class Player final : public Creature, public Cylinder
 
 	uint32_t getClientIcons() const;
 
-	const GuildWarVector& getGuildWarVector() const { return guildWarVector; }
+	const std::vector<uint32_t>& getGuildWarVector() const { return guildWarVector; }
```

Comment:
Just return `auto`. I would go further and change it to `getGuildWars` in a separate PR, we're not Hungarians